src/components/aboutus/index.jsx

```
@@ -87,7 +87,7 @@ function AboutUs() {
                                         That's where RequireSign comes in.
                                     </Typography>
 
-                                    <a href={`${try_for_free}`} target='_blank'>
+                                    <a href={`${try_for_free}`} target='_blank' style={{ textDecoration: "none" }}>
```

Biome

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)